contracts/src/TroveManager.sol

```diff
@@ -1784,6 +1785,12 @@ contract TroveManager is LiquityBase, ITroveManager, ITroveEvents {
                 if (_batchDebt == 0) {
                     batchDebtSharesDelta = debtIncrease;
                 } else {
+                    // For the shares ratio to decrease 1e9 (1e18/MIN_BATCH_SHARES_RATIO),
+                    // at an average annual rate of 10%, it would take more than 217 years (log(1e9)/log(1.1))
+                    // at an average annual rate of 50%, it would take more than 51 years (log(1e9)/log(1.5))
```

Comment:
This math is assuming no compounding, which is the best case scenario (longest time to reach the limit).

If the batch is being touched frequently, it'll be closer to:
```
t_1e9 = log(1e9) / r
```

where `r` is the annual interest rate, i.e. `0.1` & `0.5` in the examples above.

Reply:
I guess my wording was ambiguous, but I was not assuming it was without compounding. I was assuming those “global average annual rates” would include not only compounding, but also fees.
A “global average automatic annual debt increase” (I don’t know how to call it :sweat_smile: ) of 10% sustained over the years sounds pretty high to me, let alone 50%.